### PR TITLE
feat: add support for ttft

### PR DESCRIPTION
**Pull request by LastRemote**
### Related Issues

- see https://github.com/deepset-ai/haystack/pull/8444

### Proposed Changes:

Support TTFT by casting isoformat to a datetime object and include it in a generation.

### How did you test it?

Unit test. I also have E2E tests in my version, which is almost the same but not exactly.

### Checklist

- I have read the [contributors guidelines](https://github.com/deepset-ai/haystack-core-integrations/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/deepset-ai/haystack-core-integrations/blob/main/CODE_OF_CONDUCT.md)
- I have updated the related issue with new insights and changes
- I added unit tests and updated the docstrings
- I've used one of the [conventional commit types](https://www.conventionalcommits.org/en/v1.0.0/) for my PR title: `fix:`, `feat:`, `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`.

### CODE REVIEW

**Comment by vblagoje** [comment]
Hey @LastRemote it seems like you nee to sign CLA for https://github.com/deepset-ai/haystack-core-integrations - perhaps you have already? 🙏

**Comment by LastRemote** [comment]
> Hey @LastRemote it seems like you nee to sign CLA for https://github.com/deepset-ai/haystack-core-integrations - perhaps you have already? 🙏

No I actually haven't. But now it's done.

File: integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py [hunk 1]
```diff
@@ -148,7 +149,14 @@ def trace(self, operation_name: str, tags: Optional[Dict[str, Any]] = None) -> I
             replies = span._data.get("haystack.component.output", {}).get("replies")
             if replies:
                 meta = replies[0].meta
-                span._span.update(usage=meta.get("usage") or None, model=meta.get("model"))
+                completion_start_time = meta.get("completion_start_time")
+                if completion_start_time:
+                    completion_start_time = datetime.fromisoformat(completion_start_time)
```
Comment (vblagoje) [inline integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py:154]: `fromisoformat` throws ValueError for irregular format provided. Let's just log it here and skip ttft in that case, wdyt?

**Review by vblagoje — CHANGES_REQUESTED** [review]
@LastRemote looks good but let's just handle this ValueError in case the time data is invalid

File: integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py (hunk 1 above)
Comment (LastRemote) [reply integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py:154]: Sure, will do. Normally this won't be the case though.

Comment (vblagoje) [reply integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py:154]: Yeah I agree, thanks for your understanding 👍

Comment (LastRemote) [reply integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py]: Updated, and also added a new unit test for this.

**Review by vblagoje — APPROVED** [review]
Looks good @LastRemote - thanks for another great contribution!

**Pull request merged**